src/frontend/ts/state/chessStore.ts

```diff
@@ -106,12 +94,6 @@ const useChessStore = create<ChessState>((set, get) => ({
       quit: () => {}
     })
   },
-  //  XX: maybe there should be a new getArchivedGame func
-  //      - onClick func for archive game icon
-  //      - scry for game info
-  //      - set position accordingly
-  //      - setDisplayGame
-  //
   getArchive: async (data: ArchivedGameInfo) => {
     const moves = await scryMoves('chess', '/game/' + data.gameID + '/moves');
```

Comment:
new `getArchive()` scries for an array of moves on the new `/game/@ta/moves` scry path, puts those moves into what `ArchivedGameInfo` is passed in, and then displays the game.